src/analyses/apron/apronPriv.apron.ml

```diff
@@ -404,6 +403,7 @@ struct
     {apr = getg (); priv = startstate ()}
 
   let iter_sys_vars getg vq vf = () (* TODO: or report singleton global for any Global query? *)
+  let invariant_vars ask getg st = protected_vars ask (* TODO: is this right? *)
```

Comment:
I am not very confident here either, but I don't think we ended up using this privatization much anyway...
And if we wanted to, we would need to carefully re-check it either way.

Reply:
Given that it is named protection-based, I guess there's not many other possibilities. Anyway, there's no _wrong_ implementations really: it just means either too few or too many invariants are emitted, but they're all emitted soundly via `read_global`.